src/utils/querybuilder.util.js

```diff
@@ -19,7 +19,11 @@ const stringQueryBuilder = function ({ target }) {
     if (typeof target !== 'string') {
         return {};
     }
-    const t2 = target.replace(/[\\(\\)\\-\\_\\+\\=\\/\\.]/g, '\\$&');
+    let t2 = target.replace(/[\\(\\)\\-\\_\\+\\=\\/\\.]/g, '\\$&');
+    if (t2.startsWith('contains:')) {
+        t2 = t2.substring(9);
+        return { $regex: new RegExp(escapeRegExp(t2), 'i') };
+    }
```

Comment:
We can handle contains case using FilterByMissing class (src/operations/query/r4.js, line 114) used by REST instead of handling separately for graphql

Reply:
modified to use REST path